### ISSUE TITLE
Multiple:true might work unexpected

### ISSUE THREAD

**Issue by okadurin** [tonytomov/jqGrid#681]
A column with the "edittype select" and the "multiple: true" might work unexpected.
I faced with this issue when sent a data string for editoptions.value using a wrong method. Instead of calling  "resp.getWriter().print(data)" I called "resp.getWriter().println(data)" on the server side. And it put a line break symbol at the ending. 

Steps to reproduce an issue.
1. Click on a cell and choose the "label1" and the "label3". Press Enter.
1.1. Click on the cell. Notice that only "label1" is selected but "label3" isn't.
1.2. Press Esc (like you want to cancel editing). Notice that only "label1" is displayed! We've lost the "label3"!
2. Click on the cell and choose the "label3" only (like you want to change the value). Press Enter.
2.1. Click on the cell. Notice that no item is selected! //"label3" is supposed to be selected.
2.2. Press Esc (like you want to cancel editing). Notice that no items are displayed! //"label3" is supposed to be displayed.

==================================
A jsFiddle to reproduce steps: http://jsfiddle.net/ovkadurin/teos9a50/. It used jqGrid 4.6 but 4.7 has the same issue.

==================================
SOLUTION: This [line of code]( https://github.com/tonytomov/jqGrid/blob/master/js/jquery.jqGrid.js#L13191):
```javascript
if($.inArray(sv[1],scell)>-1) {
```
should use $.trim as follows:
```javascript
if($.inArray($.trim(sv[1]),scell)>-1) {
```
like [you do it](https://github.com/tonytomov/jqGrid/blob/master/js/jquery.jqGrid.js#L13182) for the "cell" variable.

**Comment by tonytomov** [comment]
Hello,
Many thanks for this description of the problem with the recommended fix. I will publish the fix tomorrow.

Best Regards

**Comment by tonytomov** [comment]
Fixed in 4.8 release
